src/main/java/org/jabref/gui/filelist/LinkedFilesEditDialogViewModel.java

```diff
@@ -128,4 +128,12 @@ public LinkedFile getNewLinkedFile() {
         return new LinkedFile(description.getValue(), link.getValue(), monadicSelectedExternalFileType.map(ExternalFileType::toString).getOrElse(""));
     }
 
+    private void relativizeFile() {
```

Comment:
Right now, it is hard to tell from the outside on which path this methods operates. I would thus prefer if `relativizeFile` would accept a path and return the relativized path (probably also rename to `relativize[Path]`). Then in would look like `link.set(relativize(path))`.

Reply:
I already had considered this option as well, but seem to forget about it ;)